ops/terraform/locals.tf

```diff
@@ -18,6 +19,7 @@ locals {
       appsubnetcidr = "10.1.1.0/24"
       websubnetcidr = "10.1.2.0/24"
       lbsubnetcidr  = "10.1.3.0/24"
+      dbsubnetcidr  = "10.0.4.0/24"
```

Comment:
just notice this as well. this cidr will probably need to be `10.1.4.0/24` and not `10.0.4.0/24` to eliminate the change of crossing on top of each other when both environments are running.